app/survey_config/business_config.py

```diff
@@ -30,10 +36,10 @@ class BusinessSurveyConfig(
 
     footer_legal_links: Iterable[Mapping] = field(
         default_factory=lambda: [
-            Link(lazy_gettext("Cookies"), "#").__dict__,
+            Link(lazy_gettext("Cookies"), f"{base_url}/cookies").__dict__,
             Link(
                 lazy_gettext("Privacy and data protection"),
-                "#",
+                f"{base_url}/privacy-and-data-protection",
```

Comment:
Same here, the URL should come from the property we already created instead of reconstructing.

Reply:
Changed, thanks!